qa/integration/fixtures/certificates/certificate.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
If there's a script to generate this crt+key, do we need to have the output (crt+key) files checked in?